18 Normalizr/src/pages/student-detail/student.validation.ts

```diff
@@ -26,6 +28,11 @@ class StudentFormValidation extends BaseFormValidation {
       "email",
       emailValidationHandler,
     );
+
+    this._validationEngine.addFieldValidation(
+      "country",
+      requiredIdValidationHandler,
```

Comment:
I'm liking this validation engine, need some evolving and perhaps some better names (handler?) but it's looking good.